src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonViewModel.cs

```diff
@@ -16,7 +16,7 @@ namespace NexusMods.App.UI.Controls.Spine.Buttons.Download;
 [UsedImplicitly]
 public class SpineDownloadButtonViewModel : AViewModel<ISpineDownloadButtonViewModel>, ISpineDownloadButtonViewModel
 {
-    private const int PollTimeMilliseconds = 1000;
+    private const int PollTimeMilliseconds = 100;
```

Comment:
Do we need the higher resolution?

Reply:
I think this might actually be counterproductive, as if you have a larger time span, the data is less susceptible to fluctuation, is more stable and closer to the average.